source/dotnet/Library/AdaptiveCards.Rendering.Wpf/Helpers/SelectActionHelper.cs

```diff
@@ -34,19 +34,7 @@ public static FrameworkElement RenderSelectAction(this AdaptiveRenderContext con
                 uiButton.Content = uiElement;
                 uiButton.Style = context.GetStyle("Adaptive.Action.Tap");
 
-                // Handle ShowCard
-                if (selectAction is AdaptiveShowCardAction showCardAction)
-                {
-                    var actionsConfig = context.Config.Actions;
-                    bool isInline = (actionsConfig.ShowCard.ActionMode == ShowCardActionMode.Inline);
-                    if (isInline && context.CardDepth == 1)
-                    {
-                        FrameworkElement uiShowCardContainer = showCardAction.CreateShowCard(context, actionsConfig);
-
-                        // Add to the list of show cards in context
-                        context.ActionShowCards.Add(new Tuple<FrameworkElement, Button>(uiShowCardContainer, uiButton));
-                    }
-                }
+                // SelectAction doesn't allow showCard actions
```

Comment:
>// SelectAction doesn't allow showCard actions [](start = 16, length = 46)

What is the behavior here after this change? Will the host still be called back to display the show card as a popup?

Reply:
The previous version allowed select actions to have showCard actions, that isnt allowed in any of the other renderers, this section used to set the default behaviour for inline show card which shouldn't be here

---
In reply to: [272645334](https://github.com/Microsoft/AdaptiveCards/pull/2625#discussion_r272645334) [](ancestors = 272645334)

Reply:
Show cards on select actions works in the designer, so it should probably work everywhere else. It works in UWP only if host config is set to popup, which i logged a bug for.

---
In reply to: [273187558](https://github.com/Microsoft/AdaptiveCards/pull/2625#discussion_r273187558) [](ancestors = 273187558,272645334)

Reply:
This card works in the javascript visualizer, the show card shows as a popup.
[https://adaptivecards.io/visualizer/index.html](https://adaptivecards.io/visualizer/index.html)

{
    "type": "AdaptiveCard",
    "body": [
        {
            "type": "Container",
            "selectAction": {
                "type": "Action.ShowCard",
                "card": {
                    "type": "AdaptiveCard",
                    "$schema": "[http://adaptivecards.io/schemas/adaptive-card.json](http://adaptivecards.io/schemas/adaptive-card.json)",
                    "body":[
                        {
                            "type": "TextBlock",
                            "text":"Foo"
                        }
                    ]
                }
            },
            "items": [
                {
                    "type": "TextBlock",
                    "text": "New TextBlock"
                }
            ]
        }
    ],
    "$schema": "[http://adaptivecards.io/schemas/adaptive-card.json](http://adaptivecards.io/schemas/adaptive-card.json)",
    "version": "1.0"
}

---
In reply to: [273209963](https://github.com/Microsoft/AdaptiveCards/pull/2625#discussion_r273209963) [](ancestors = 273209963,273187558,272645334)

Reply:
I have this from the documentation

[https://adaptivecards.io/explorer/Image.html](https://adaptivecards.io/explorer/Image.html)
[https://adaptivecards.io/explorer/ColumnSet.html](https://adaptivecards.io/explorer/ColumnSet.html)
[https://adaptivecards.io/explorer/Column.html](https://adaptivecards.io/explorer/Column.html)

---
In reply to: [273210500](https://github.com/Microsoft/AdaptiveCards/pull/2625#discussion_r273210500) [](ancestors = 273210500,273209963,273187558,272645334)

Reply:
@matthidinger @andrewleader Should show cards work for select actions? Documentation says no, but current behavior in JS is that they work, and the designer explicitly lets you add them, although they are always popups regardless of host config settings. Behavior on UWP is that they work if host config is set to popup and are dropped (with a warning) otherwise. Not sure about the other platforms. (I logged #2631 for UWP because i assumed js was the correct behavior)

---
In reply to: [273223131](https://github.com/Microsoft/AdaptiveCards/pull/2625#discussion_r273223131) [](ancestors = 273223131,273210500,273209963,273187558,272645334)

Reply:
There was some discussion about this and the behaviour was agreed on not allowing show card actions and filing a warning 

---
In reply to: [273232439](https://github.com/Microsoft/AdaptiveCards/pull/2625#discussion_r273232439) [](ancestors = 273232439,273223131,273210500,273209963,273187558,272645334)